pkg/chaosctl/cmd/debug.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2020 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	"github.com/chaos-mesh/chaos-mesh/pkg/debug/iochaos"
+	"github.com/chaos-mesh/chaos-mesh/pkg/debug/networkchaos"
+	"github.com/chaos-mesh/chaos-mesh/pkg/debug/stresschaos"
+)
+
+type DebugOptions struct {
+	ChaosName string
+	Namespace string
+}
```

Comment:
I feel like the k8s client should be part of this method and we can pass it to the debug method.

Reply:
Thx I create the clientset (one for controller-runtime and another for go-client, since I need them both) in one place and send them everywhere as a parameter. 
https://github.com/chaos-mesh/chaos-mesh/blob/26eaa3a19746eb9698b61627e7a540ffc8d4b8ee/pkg/debug/common/common.go#L55-L58
https://github.com/chaos-mesh/chaos-mesh/blob/26eaa3a19746eb9698b61627e7a540ffc8d4b8ee/pkg/debug/common/common.go#L203-L208
It's just a bit ugly ... do you have some suggestions on this